nntrainer/src/tensor.cpp

```diff
@@ -248,7 +248,6 @@ void Tensor::saxpy(const unsigned int N, const float alpha, const float *X,
 #ifdef USE_BLAS
   cblas_saxpy(N, alpha, X, incX, Y, incY);
 #else
-  unsigned int xi, yi;
   if (incX <= 0 or incY <= 0)
```

Comment:
I think this can be simplified to just call saxpy from blas_interface?

Reply:
fixed. thanks.